### PR TITLE
PlayerController refactor, cleaned up console errors and logs 

### PR DESCRIPTION
**Pull request by runningjump**
GitHib is doing an awful job at diffing PlayerController, but mainly I just made a generic Collider2D property so we can reference the collider everywhere (Enable and Disable) without needing to do a switch on the playerType every time.

Also moved around some stuff in PlayerController into groups, added some comments, and removed what looked unused to me. Let me know if I removed anything that you did actually have plans for...

Aside from that file, I just went down the console error rabbit hole and cleaned up all up. So we should have no more errors and a lot fewer messages! Should make debugging new things easier.

### CODE REVIEW

File: Assets/Scripts/PlayerController.cs [hunk 1]
```diff
@@ -1,644 +1,544 @@
...
+    private bool canMove = true;
+
+    // Particle system
+	public ParticleSystem ps;
+
+    // Rigidbody, mesh, colliders
+    Rigidbody2D rb;
+    MeshRenderer mr;
+    public PolygonCollider2D trianglePC, trapezoidPC, rectanglePC, starPC;
+    private Collider2D shapeCollider;
```
Comment (nicksantan) [inline Assets/Scripts/PlayerController.cs:44]: Cool, I wasn't aware of this generic 'Collider2D' type.

File: Assets/Scripts/PlayerController.cs [hunk 2]
```diff
@@ -1,644 +1,544 @@
...
-		
-		}
-	
-	}
-}
+        // Get collider for chosen shape
+        shapeCollider.enabled = true;
+
+        // Push play event
+        string shapePlayEvent = playerShape + "Plays";
```
Comment (nicksantan) [inline Assets/Scripts/PlayerController.cs:170]: Nice, I see how this kind of consolidating is making stuff like this simpler and less 'if / else' or 'switch' block-y

File: Assets/Scripts/PlayerController.cs [hunk 3]
```diff
@@ -1,644 +1,544 @@
...
+    private float speedPowerupTimer;
+    private bool speedPowerupActive = false;
+
+    private float sizePowerupTimer;
+    private bool sizePowerupActive = false;
+
+    private float pandemoniumTimer;
+    private bool pandemoniumPowerupActive = false;
+
+    // TODO Get audio clips from global object?
```
Comment (nicksantan) [inline Assets/Scripts/PlayerController.cs:64]: Definitely a good idea! Assigning these sounds to every player object when one changes is a pain.

File: Assets/Scripts/PlayerController.cs (hunk 1 above)
Comment (runningjump) [reply Assets/Scripts/PlayerController.cs:44]: Yeah! PolygonCollider2D, CircleCollider2D and BoxCollider2D all inherit from it, so you can cast them as a Collider2D variable and call their shared functions without the compiler complaining.

File: Assets/Scripts/PlayerController.cs (hunk 3 above)
Comment (runningjump) [reply Assets/Scripts/PlayerController.cs:64]: Are these in the AudioManager already?

Comment (nicksantan) [reply Assets/Scripts/PlayerController.cs:64]: No I don't think they are --

**Pull request merged**